dev/pastefromword/getclipboard.html

```diff
@@ -42,15 +49,48 @@
 		if ( evt.data.dataTransfer && evt.data.dataTransfer.getData( 'text/html', true ) ) {
 			val = evt.data.dataTransfer.getData( 'text/html', true );
 		}
+		document.getElementById( 'rawHtml' ).value = val;
+
+		if ( evt.data.dataTransfer && evt.data.dataTransfer.getData( 'text/rtf' ) ) {
```

Comment:
The `val` is not erased which means when there is no RTF data it will show HTML inside RTF block (I see it worked like that before, looks like we missed it earlier). `val` should be emptied between reads. You may also consider adding some message or changing container border color if content is empty.